examples/redaction-ruleset-project/src/main/ml-schemas/redactionRules/permissions.properties

```diff
@@ -0,0 +1,5 @@
+# Add one or more document permissions to apply to the ruleset
```

Comment:
I think it's unlikely that permissions will need to be different per ruleset, while it's very likely that collections will need to be different. I don't want to give the impression that this file is needed, as it's most likely not - so remove it from the example, just keep the collections example (which is a very good one).